docs/Other SDL Features/Remote Control Vehicle Features/index.md

```diff
@@ -234,23 +374,46 @@ sdlManager.sendRPC(interiorVehicleData);
+
+The permission area of a module depends on the `serviceArea`s which can be found in `moduleInfo`. Each module will have its own `location` and `serviceArea`. The `location` of a module is just like the `seats` grid, it tells you the physical location of a particular module. The `serviceArea` tells you how far that module reaches. A radio control serves all passengers in the vehicle. Its service area will cover its physical location as well as other locations in the vehicle. If a `serviceArea` is not included, it is assumed the `serviceArea` is the same as `location`. If both are not included, it is assumed that the `serviceArea` covers the whole area of the vehicle. If a user is not sitting within the `serviceArea`s `grid` they will not be able to control that module.
 
 @![iOS]
 ##### Objective-C
+###### No ModuleID
 ```objc
 SDLTemperature *temperature = [[SDLTemperature alloc] initWithUnit:SDLTemperatureUnitFahrenheit value:74.1];
 SDLClimateControlData *climateControlData = [[SDLClimateControlData alloc] initWithFanSpeed:@2 desiredTemperature:temperature acEnable:@YES circulateAirEnable:@NO autoModeEnable:@NO defrostZone:nil dualModeEnable:@NO acMaxEnable:@NO ventilationMode:SDLVentilationModeLower heatedSteeringWheelEnable:@YES heatedWindshieldEnable:@YES heatedRearWindowEnable:@YES heatedMirrorsEnable:@NO];
 SDLModuleData *moduleData = [[SDLModuleData alloc] initWithClimateControlData:climateControlData];
 SDLSetInteriorVehicleData *setInteriorVehicleData = [[SDLSetInteriorVehicleData alloc] initWithModuleData:moduleData];
 [self.sdlManager sendRequest:setInteriorVehicleData];
 ```
+###### With ModuleID
+```objc
+SDLTemperature *temperature = [[SDLTemperature alloc] initWithUnit:SDLTemperatureUnitFahrenheit value:74.1];
+SDLClimateControlData *climateControlData = [[SDLClimateControlData alloc] initWithFanSpeed:@2 desiredTemperature:temperature acEnable:@YES circulateAirEnable:@NO autoModeEnable:@NO defrostZone:nil dualModeEnable:@NO acMaxEnable:@NO ventilationMode:SDLVentilationModeLower heatedSteeringWheelEnable:@YES heatedWindshieldEnable:@YES heatedRearWindowEnable:@YES heatedMirrorsEnable:@NO];
+SDLModuleData *moduleData = [[SDLModuleData alloc] initWithClimateControlData:climateControlData];
+moduleData.moduleId = <#Module ID#>
+SDLSetInteriorVehicleData *setInteriorVehicleData = [[SDLSetInteriorVehicleData alloc] initWithModuleData:moduleData];
+[self.sdlManager sendRequest:setInteriorVehicleData];
```

Comment:
All of our example code needs the `withResponseHandler` at least checking the success of the RPC.

Reply:
fixed

Reply:
This was not fixed...